OPHD/UI/NotificationArea.h

```diff
@@ -65,10 +65,13 @@ class NotificationArea : public Control
 	void updateRectListPositions();
 
 	const NAS2D::Image& mIcons;
+	const NAS2D::Font& mFont;
+
 	std::vector<Notification> mNotificationList;
 	std::vector<NAS2D::Rectangle<int>> mNotificationRectList;
 
 	std::size_t mNotificationIndex{ SIZE_MAX };
+	NAS2D::Rectangle<int> mNotificationBriefRect{ 0 };

```

Comment:
You can initialize using empty `{}`. That might be less confusing, as here there is a 4 element struct, where only 1 element is listed and 3 are omitted. Specifying one element kind of makes this look like it's initializing a scalar type, which it's not.
```cpp
NAS2D::Rectangle<int> mNotificationBriefRect{};
```

----

If you want to get pedantic about how the standard treats this, using C++ legalese:

Using `{}` on a struct causes [aggregate initialization](https://en.cppreference.com/w/cpp/language/aggregate_initialization). When fewer parameters are given than struct elements, this leads to value initialization of the unspecified elements.
> If the number of initializer clauses is less than the number of members or initializer list is completely empty, the remaining members are value-initialized.

When [value initialization](https://en.cppreference.com/w/cpp/language/value_initialization) is done on the element type, which in this case is the primitive type `int`, which is not a class type, this translates to zero initialization.
> 4) otherwise, the object is zero-initialized.

When [zero initialization](https://en.cppreference.com/w/cpp/language/zero_initialization) is done, the element values are set to zero of the corresponding type.
> If T is a scalar type, the object's initial value is the integral constant zero explicitly converted to T.

So according to the requirements of the C++ standard, `{}` applied to a struct of `int` values will set each `int` value to `0`.

Reply:
I figured a single 0 would fill the entire structure with 0 -- I'm still getting used to the `{}` initializer syntax and making assumptions based on the 'C' part of the language.